release.yml

```diff
@@ -8,7 +8,7 @@ trigger:
 pr:
 - dotliquid
 - main
-- xoriant/*
+- templates/*
```

Comment:
It looks we already have this change in `microsoft:templates/m2` ([ref](https://github.com/microsoft/FHIR-Converter/blob/1268ed7913fbd210fa5b3dd336899f48f4de962b/release.yml#L11)). Are we using the latest code from `microsoft:templates/m2`?

Reply:
Yes , we are using the latest code from microsoft:templates/m2